XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Hands/BaseHand.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Devices;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.InputSystem;
+using XRTK.Interfaces.Providers.Controllers;
+
+namespace XRTK.Providers.Controllers.Hands
+{
+    public abstract class BaseHand : BaseController, IMixedRealityHand
+    {
+        // Hand ray
+        protected HandRay HandRay { get; } = new HandRay();
+
+        public override bool IsInPointingPose
```

Comment:
`IsInPointingPose` is nowhere defined in XRTK to override. Not sure really how to resolve this. @StephenHodgson maybe?

Reply:
There was a comment raise about pointing pose that people don't like it.  We should look to review this implementation.

Reply:
What about implementing `IsInPointingPose` in `IMixedRealityHand`?